community-membership.md

```diff
@@ -1,63 +1,99 @@
-| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
-| Subproject owner | Set direction and priorities for a subproject | Demonstrated responsibility and excellent technical judgement for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
-
-## New contributors
-
-[New contributors] should be welcomed to the community by existing members,
-helped with PR workflow, and directed to relevant documentation and
-communication channels.
-
-## Member
-
-Members are continuously active contributors in the community.  They can have
-issues and PRs assigned to them, participate in SIGs through GitHub teams, and
-pre-submit tests are automatically run for their PRs. Members are expected to
-remain active contributors to the community.
-
-**Defined by:** Member of the milvus-io GitHub organization
-
-## Reviewer
-
-Reviewers are able to review code for quality and correctness on some part of a
-subproject. They are knowledgeable about both the codebase and software
-engineering principles.
-
-**Defined by:** *reviewers* entry in an OWNERS file in a repo owned by the
-milvus project.
-
-## Approver
-
-Code approvers are able to both review and approve code contributions.  While
-code review is focused on code quality and correctness, approval is focused on
-holistic acceptance of a contribution including: backwards / forwards
-compatibility, adhering to API and flag conventions, subtle performance and
-correctness issues, interactions with other parts of the system, etc.
-
-**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the
-milvus project.
-
-## Subproject Owner
-
-**Note:** This is a generalized high-level description of the role, and the
-specifics of the subproject owner role's responsibilities and related
-processes *MUST* be defined for individual SIGs or subprojects.
-
-Subproject Owners are the technical authority for a subproject in the Kubernetes
-project.  They *MUST* have demonstrated both good judgement and responsibility
-towards the health of that subproject.  Subproject Owners *MUST* set technical
-direction and make or approve design decisions for their subproject - either
-directly or through delegation of these responsibilities.
-
-**Defined by:** *owners* entry in subproject [OWNERS] files as defined by [sigs.yaml]  *subproject.owners*
-
-
+| Contributor | Active contributor in the community | Finish at least one contribution to the project | Github all contributor list|
+| Committer | Review contributions from other members | Finish at least one major contributions, History of reviews| [COMITTER](https://github.com/milvus-io/milvus/blob/master/OWNERS) |
+| Maintainer | Contributions acceptance approval| Highly experienced active reviewer and contributor| [MAINTAINERS](https://github.com/milvus-io/milvus/blob/master/OWNERS_ALIASES)|
```

Comment:
I'm a little confused. How do you define Highly experienced active reviewer and contributor？ And what is the "reviewer" role?

Reply:
I think "committer" is the role you mentioned as "reviewer" role. 
I read the essay https://opensource.guide/leadership-and-governance/ 
A “contributor” could be anyone who comments on an issue or pull request, people who add value to the project (whether it’s triaging issues, writing code, or organizing events), or anybody with a merged pull request (perhaps the narrowest definition of a contributor).
The term “committer” might be used to distinguish commit access, which is a specific type of responsibility, from other forms of contribution.
It seems that we follow the similar definition, right? @xiaofan-luan